src/lib/structures/fpa_rounding.mli

```diff
@@ -40,13 +42,20 @@ type rounding_mode =
   | Up
   | Down
   | NearestTiesToAway
+[@@deriving ord]
 
 (** Equal to ["RoundingMode"], the SMT2 type of rounding modes. *)
 val fpa_rounding_mode_type_name : string
 
 (** Equal to ["fpa_rounding_mode"], the Alt-Ergo native rounding mode type. *)
 val fpa_rounding_mode_ae_type_name : string
 
+(** The Dolmen rounding mode type. *)
+val fpa_rounding_mode_dty : Dolmen.Std.Expr.Ty.t
```

Comment:
:+1: 

Do we still need the various string conversion functions with this?

Reply:
We still need them in both legacy and new frontend. In the new frontend, we use these functions in the typechecker to inject the AE identifiers into SMT-LIB identifiers. 

We also use them in `Expr` because we need to convert a `Hstring` identifier into `Fpa_rounding.rounding_mode`. 

I'm sure we can some of them after getting rid of the legacy frontend.